pkg/apis/utils/resources/resources.go

```diff
@@ -0,0 +1,72 @@
+package resources
+
+import (
+	"github.com/rs/zerolog/log"
+	corev1 "k8s.io/api/core/v1"
+	resourcev1 "k8s.io/apimachinery/pkg/api/resource"
```

Comment:
```suggestion
	"k8s.io/apimachinery/pkg/api/resource"
```
resource is not versioned

Reply:
fixed